src/User32.Desktop/User32+DISPLAYCONFIG_SOURCE_MODE_INFO.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DISPLAYCONFIG_SOURCE_MODE_INFO"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct DISPLAYCONFIG_SOURCE_MODE_INFO
+        {
+            private uint bitvector;
+
+            public uint CloneGroupId
+            {
+                get
+                {
+                    return this.bitvector & 0xFFFF;
+                }
+
+                set
+                {
+                    this.bitvector = value | this.bitvector;
```

Comment:
This and other such property setters you've defined are buggy. Consider: it's impossible to set the value to zero because it is always OR'd with its existing value.